stl/inc/type_traits

```diff
@@ -2082,6 +2092,36 @@ struct _Is_trivially_swappable : bool_constant<_Is_trivially_swappable_v<_Ty>> {
     // true_type if and only if it is valid to swap two _Ty lvalues by exchanging object representations.
 };
 
+#ifdef __cpp_lib_concepts // TRANSITION, GH-395
+_EXPORT_STD template <class _From, class _To>
+concept convertible_to =
+#if !defined(__EDG__) && !defined(__clang__) // TRANSITION, DevCom-1627396
+    is_convertible_v<_From, _To>
+#else // ^^^ workaround / no workaround vvv
+    __is_convertible_to(_From, _To)
+#endif // ^^^ no workaround ^^^
+    && requires { static_cast<_To>(_STD declval<_From>()); };
+
+template <class _RefWrap, class _Ty, class _RefWrapQ, class _TyQ>
+concept _Has_ref_wrap_common_reference = _Is_specialization_v<_RefWrap, reference_wrapper>

```

Comment:
Why change the name of the concept?

Reply:
Oh, I used to consider combining it with other stuffs, but gave up later. Reverting to the "original" name.